public/sw.js

```diff
@@ -1,6 +1,7 @@
-/* eslint-disable no-loop-func */
 /* eslint-disable no-undef */
 /* eslint-disable no-restricted-globals */
+/* eslint-disable no-unused-vars */
```

Comment:
Very true and I have a good reason for doing so. I added those comments in the end after I coded everything and checked to make sure that there were no unused variables. The service worker only imports scripts. Importing scripts and importing files like you regularly would in a component are not the same. When you import a script, you're importing both the script's functions and global variables. In the service worker, I imported 3 scripts which makes them connect all together as one. Due to this, I could define a global variable in one script and call it in another even if it's not defined in that other script. Thanks to Dr. Tuck who helped me find that out, I took advantage of that to reduce the number of variables I would have to create. The extension ES-Lint doesn't really like that at all and screams at me for doing so 😂 So I had to put those comments there for each script so that the code would pass in Travis-CI